phpstan.neon

```diff
@@ -6,9 +6,15 @@ parameters:
     - %currentWorkingDirectory%/appinfo/Migrations/*.php
     - %currentWorkingDirectory%/appinfo/routes.php
   ignoreErrors:
+    -
+      message: '#^Right side of || is always false.$#'
+      path: lib/Controller/DocumentController.php
+      count: 1
```

Comment:
This is a new thing that phpstan is finding.
IMO the code can be easily and safely fixed, but actually the reported problem does no harm, so I won't touch real code in this PR.